src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -364,6 +364,15 @@ function InitializeFirebaseAdminStubs(frb: FunctionsRuntimeBundle): typeof admin
 
   let hasInitializedSettings = false;
   const initializeSettings = (app: admin.app.App, userSettings: any) => {
+    const isEnabled = isFeatureEnabled(frb, "admin_stubs");
```

Comment:
Why not just block out this whole function if `admin_stubs` is not enabled?

Reply:
Because we still need a reference to the function's admin app to generate snapshots, so stubbing this all out would require us to add more code to check if the dev has created a default app yet. I'm not sure which is better, but this is fewer code changes for now.